wsgidav/debug_filter.py

```diff
@@ -99,12 +99,13 @@ def __call__(self, environ, start_response):
             dumpRequest = dumpResponse = True
 
         # Process URL commands
-        if "dump_storage" in environ.get("QUERY_STRING"):
-            dav = environ.get("wsgidav.provider")
-            if dav.lockManager:
-                dav.lockManager._dump()
-            if dav.propManager:
-                dav.propManager._dump()
+        if environ.get("QUERY_STRING"):
+            if "dump_storage" in environ.get("QUERY_STRING"):
```

Comment:
I think it would be easier to simply change that line to 

    if "dump_storage" in environ.get("QUERY_STRING", ""):

instead?